packages/buckets/src/api/index.ts

```diff
@@ -0,0 +1,523 @@
+import log from 'loglevel'
```

Comment:
i'm expecting that i could `import { specifics } from '@textile/buckets/api'`, so this has no crypto. however... that always confuses me a bit. so maybe that dep would show up unless i make this a standalone package...

Reply:
let me know your thoughts here @carsonfarmer